search/searcher/search_term_prefix.go

```diff
@@ -55,6 +62,8 @@ func NewTermPrefixSearcher(indexReader index.IndexReader, prefix string, field s
 		return nil, err
 	}
 
+	qsearchers = nil
```

Comment:
Ah, in that case i don't understand the change at all.  The qsearchers should not be closed on normal exit of the function, the live, non-closed searchers are to be used by the disjunction searcher.